### ISSUE TITLE
fit_features_together - Interpretation

### ISSUE THREAD

**Issue by dataversenomad** [MobileTeleSystems/RecTools#147]
### Your Question

Hi team.

I am currently using your package, and it has helped me a lot. Now, I have one doubt regarding the 'fit_features_together' parameter.

Let's say I have use the constructor:

dataset_user_features = Dataset.construct(
    interactions_df=train,
    user_features_df=user_features,
    cat_user_features = user_categorical_features,
)

Now, if I set the parameter fit_features_together = False, Does it mean that the model will completely avoid integrating the 'user_features' ? 

I have read the docs, however I am still in doubt. Explicitly, I am using the ImplicitALSWrapperModel.

I really appreciate all your help.

### RecTools version

rectools==0.5.0

**Comment by feldlime** [comment]
Yeah, I agree, we haven't explained that moment in docs. 

TL;DR No, your features will be used anyway. `fit_features_together` controls the way of training model. You can try both `True` and `False` and compare the quality.  
For more info check this [example](https://github.com/MobileTeleSystems/RecTools/blob/main/examples/5_benchmark_iALS_with_features.ipynb), hopefully it would help.

**Comment by dataversenomad** [comment]
Awesome! Thank you very much.

**Comment by feldlime** [comment]
Happy to help!
